### PR TITLE
Add setmode() and gettime() ordinals

### PR DESCRIPTION
**Pull request by tuday2**
Still need to write tests

### CODE REVIEW

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs [hunk 1]
```diff
@@ -7868,5 +7874,65 @@ private void txtvars_delegate()
...
+                Registers.AX = 0xFFFF; // -1
+                return;
+            }
+
+            var fileStructPointer = Module.Memory.GetOrAllocateVariablePointer($"FILE_{fileStreamPointer.Name}-{FilePointerDictionary.Count}", FileStruct.Size);
+            var fileStruct = new FileStruct(Module.Memory.GetArray(fileStructPointer, FileStruct.Size));
+
+            switch (fileMode)
+            {
+                case 0x4000: //TEXT
```
Comment (paladine) [inline MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:7905]: in theory a client could send a bitmask with both TEXT & BINARY set, but it's probably such a fringe case. Maybe we could throw an exception?

File: MBBSEmu.Tests/ExportedModules/Majorbbs/gettime_Tests.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            var dosTimeDateStructPointer = mbbsEmuMemoryCore.AllocateVariable("DOSDATETIME", TimeStruct.Size);
+            
+            //Execute Test
+            ExecuteApiTest(HostProcess.ExportedModules.Majorbbs.Segment, GETTIME_ORDINAL, new List<FarPtr> { dosTimeDateStructPointer });
+
+            //Verify Results
+            var dosTimeDateStruct = mbbsEmuMemoryCore.GetArray(dosTimeDateStructPointer, TimeStruct.Size);
+            
+            Assert.Equal(minute, dosTimeDateStruct[0]);
+            Assert.Equal(hour, dosTimeDateStruct[1]);
```
Comment (paladine) [inline MBBSEmu.Tests/ExportedModules/Majorbbs/gettime_Tests.cs:38]: probably want to assert on 0 == dosTimeDateStruct[2]

Comment (paladine) [reply MBBSEmu.Tests/ExportedModules/Majorbbs/gettime_Tests.cs:38]: unless of could we want to implement this using milliseconds / 10

**Review by paladine — CHANGES_REQUESTED** [review]

File: MBBSEmu.Tests/ExportedModules/Majorbbs/gettime_Tests.cs (hunk 2 above)
Comment (tuday2) [reply MBBSEmu.Tests/ExportedModules/Majorbbs/gettime_Tests.cs:38]: We'll go with 0 :)

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs (hunk 1 above)
Comment (tuday2) [reply MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:7877]: Wouldn't the default: in the switch catch that exception? 
```
                default:
                    throw new Exception($"Unknown setmode: {fileMode}");
```

**Review by paladine — APPROVED** [review]

**Pull request merged**